File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        map = {}
        
        for index,value in enumerate(nums):
            num = target-value
            if (num) in map:
                return [map[num],index]
            map[value] = index 
        return
            
        

File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int sums = 0;
        for(int i = 0; i < nums.size(); i++){
            for(int j = i+1; j < nums.size(); j++){
                if (nums[j] + nums[i] == target){
                    return {i,j};
                }
            }
        }
    return {};
    }
};